book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -52,26 +52,26 @@ $ git commit --amend --no-edit
 
-To modify a commit that is farther back in your history, you must move to more complex tools.
-Git doesn’t have a modify-history tool, but you can use the rebase tool to rebase a series of commits onto the HEAD they were originally based on instead of moving them to another one.
-With the interactive rebase tool, you can then stop after each commit you want to modify and change the message, add files, or do whatever you wish.
-You can run rebase interactively by adding the `-i` option to `git rebase`.
-You must indicate how far back you want to rewrite commits by telling the command which commit to rebase onto.
+Upang baguhin ang isang commit na iyon na mas malayo sa iyong kasaysayan, ikaw ay dapat lumipat sa higit na kumplikadong mga kasangkapan.
+Ang Git ay walang pagbabago-kasaysayan na kasangkapan, ngunit maaari mong gamitin ang rebase na kasankapan sa mga serye ng mga commit papunta sa HEAD sila ay orihinal batay sa halip na paglipat sa kanila na para sa isa pa.
+Gamit ang interactuve na rebase na kasangkapan, maaari mong itigil pagkatapos ng bawat commit na gusto mong baguhin at palitan ang mensahe, magdagdag ng mga file, o gumawa ng gusto mong gawin.
+Maaari mong patakbuhin ang rebase nang interactive sa pamamagitan sa pagdagdag ng `-i` na opsyon sa `git rebase`.
+Dapat mong ipahiwatig kung gaano kalayo ang gusto mong isulat muli sa mga commit sa pamamagitan sa pagsabi ng utos na kung saan ang commit ay nag-rebase nito.
 
-For example, if you want to change the last three commit messages, or any of the commit messages in that group, you supply as an argument to `git rebase -i` the parent of the last commit you want to edit, which is `HEAD~2^` or `HEAD~3`.
-It may be easier to remember the `~3` because you’re trying to edit the last three commits, but keep in mind that you’re actually designating four commits ago, the parent of the last commit you want to edit:
+Halimbawa, kung ikaw ay gustong baguhin ang huling tatlong commit na mga mensahe, o anuman sa mga commit na mensahe sa grupong iyon, punan mo yun bilang isang argumento sa `git rebase -i` at ang magulang sa huling commit na gusto mong baguhin, na kung saan ay ang `HEAD~2^` o `HEAD~3`.
+Maaaring mas madali itong matandaan ang `~3` bdahil ikaw ay sinusubukang mag-edit sa huling tatlong mga commit, ngunit panataliin sa isip na ikaw ay tunay na nagtatalaga ng apat na mga commit nung nakaraan, ang magulang sa huling commit na gusto mong i-edit:
 
 [source,console]
 ----
 $ git rebase -i HEAD~3
 ----
 
-Remember again that this is a rebasing command – every commit included in the range `HEAD~3..HEAD` will be rewritten, whether you change the message or not.
-Don’t include any commit you’ve already pushed to a central server – doing so will confuse other developers by providing an alternate version of the same change.
+Tandaan muli na ito ay isang rebase na utos – Sa bawat commit na kasama sa lawak ng `HEAD~3..HEAD` ay masusulatang muli, kung ikaw ay nagbago sa mensahe o hindi.
```

Comment:
* Tandaan muli to Tandaang muli o Muling tandaan
* Sa bawat > sa bawat

Reply:
done change